docs/source/_src_user_guide/state_propagation/environment_setup/create_bodies/create_body_settings.rst

```diff
@@ -89,12 +106,120 @@ Default settings may be overridden as follows:
 
           .. literalinclude:: /_src_snippets/simulation/environment_setup/req_create_bodies.cpp
              :language: cpp
+             
+The above works equally well if the existing environment model settings are empty. Where the above example creates a new gravity field settings object, and overrides the default setting for the Sun's gravity field with this new object. The new settings define a point-mass gravity field with a gravitational parameter of 1.32712440042 :math:`\cdot` 10 :superscript:`20` m :superscript:`3` / s :superscript:`2`.
+
+
+Modifying parameters in existing settings objects
+-------------------------------------------------
+
+Default settings may be overridden as follows:
+
+    .. tabs::
+
+         .. tab:: Python
+
+          .. toggle-header::
+             :header: Required **Show/Hide**
+
+             .. literalinclude:: /_src_snippets/simulation/environment_setup/req_create_bodies.py
+             .. literalinclude:: /_src_snippets/simulation/environment_setup/default_bodies.py
+                :language: python
+
+          .. literalinclude:: /_src_snippets/simulation/environment_setup/override_default_parameters.py
+             :language: python
+
+         .. tab:: C++
+
+          .. literalinclude:: /_src_snippets/simulation/environment_setup/req_create_bodies.cpp
+             :language: cpp
+
+Where the value of the gravitational parameter in the Sun's gravity field is changed to 1.32712440042 :math:`\cdot` 10 :superscript:`20` m :superscript:`3` / s :superscript:`2`. Functionally, this example is identical to the previous one, but it permits different kinds of modifications to be made. It allows only a *single* property of the environment model to be modified, while in the previous example, it is required that *all* properties are redefined by the user (for the point-mass gravity field, which has only one property in the settings, this point is moot). The present example therefor allows for more 'fine-grained' control of the settings, but limits the user to a modifying the properties of the settings, without providing the flexibility to modify the *type* of settings (which is allowed in the previous example).
```

Comment:
The present example ~~therefor~~ *therefore* allows for more...
...but limits the user to ~~a~~ modifying the properties of the settings...